rero_ils/modules/patrons/templates/rero_ils/patron_profile.html

```diff
@@ -62,134 +73,53 @@
   </header>
 
   <article class="tab-content">
-    <section {% if tab == 'checkouts' %}class="tab-pane show active py-2"{%else%}class="tab-pane show py-2"{% endif %} id="checkouts" role="tabpanel" aria-labelledby="checkouts-tab">
-      {% if checkouts|length > 0 %}
-      {{ build_table('checkouts', checkouts) }}
-      {% else %}
-      <p>{{ _('No loan') }}</p>
-      {% endif %}
+    <section
+      class="pl-3 pr-2 tab-pane show py-2{% if tab == 'loans' %} active{% endif %}"
+      id="loans"
+      role="tabpanel"
+      aria-labelledby="loan-tab"
+    >
+      {% include('rero_ils/_patron_profile_loans.html') %}
+    </section>
+    <section
+      class="pl-3 pr-2  tab-pane show py-2{% if tab == 'requests' %} active{% endif %}"
+      id="requests"
+      role="tabpanel"
+      aria-labelledby="request-tab"
+    >
+      {% include('rero_ils/_patron_profile_requests.html') %}
     </section>
-    <section {% if tab == 'pendings' %}class="tab-pane show active py-2"{%else%}class="tab-pane show py-2"{% endif %} id="pending" role="tabpanel" aria-labelledby="pending-tab">
-      {% if pendings|length > 0 %}
-      {{ build_table('requests', pendings) }}
-      {% else %}
-      <p>{{ _('No pending') }}</p>
-      {% endif %}
+    <section
+      class="pl-3 pr-2 tab-pane show py-2{% if tab == 'fees' %} active{% endif %}"
+      id="fees"
+      role="tabpanel"
+      aria-labelledby="fee-tab"
+    >
+      {% include('rero_ils/_patron_profile_fees.html') %}
     </section>
-    <section class="tab-pane py-2" id="history" role="tabpanel" aria-labelledby="history-tab">
-      {% if history|length > 0 %}
-      {{ build_table('history', history) }}
-      {% else %}
-      <p>{{ _('No history') }}</p>
-      {% endif %}
+    <section
+      class="pl-3 pr-2 tab-pane show py-2{% if tab == 'history' %} active{% endif %}"
+      id="history"
+      role="tabpanel"
+      aria-labelledby="history-tab"
+    >
+      {% include('rero_ils/_patron_profile_history.html') %}
     </section>
-    <section class="tab-pane py-2" id="personal-data" role="tabpanel" aria-labelledby="personal-data-tab">
+    <section
+      class="pl-3 pr-2 tab-pane show py-2{% if tab == 'personnal' %}active{% endif %}"
```

Comment:
a space is missing before `active`. At this time if `tab=="personnal"` then the class will be `py-2active` instead of `py-2` and `active`

Reply:
done